src/KeyVault/VaultSecret.php

```diff
@@ -14,12 +14,12 @@
 /**
  * Class VaultSecret.
  */
-class VaultSecret extends Vault
+class VaultSecret extends AbstractVault
 {
     /**
      * Get secret from the vault.
      *
-     * @param string $name the name of the secret in the vault
+     * @param string $name    the name of the secret in the vault
```

Comment:
Why the spaces?

Reply:
Also this seams to be the only one using parameter comments in onlines. I prefer the comment on the next line.

Reply:
Huh I think phpcs made this. I guess its for aligning the descriptions right above each other.